docs/components/table.md

```diff
@@ -66,9 +66,130 @@ Set this to `true` to disable ablity to sort.
 
 ### enableClientSort
 
-**Experimental** - set this prop to `true` to enable client side sorting if using a fetcher that returns unpaginatinated data.
+Set this prop to `true` to enable client side sorting if using a fetcher that returns unpaginatinated data.
 This functionality may be flaky.
 
+### sortHandlerFn
+
+Use a custom sort handler function to handle sorting table data for specific columns
+
+::: tip Notes
+
+1. In order for a column to use the custom sort handler function, `useSortHandlerFn` must be set to `true` in the [headers](#headers) object
+
+2. Sort handler functions can take four params:
+
+- `key`: the key of the column to be sorted
+- `prevKey`: the key of the column previously sorted
+- `sortColumnOrder`: the order in which to sort the column (`asc` or `desc`)
+- `data`: the data returned from the fetcher function response
+:::
+
+#### sortHandlerFn Example
+
+Here the `last_seen` column is set to use the custom sort handler function via the `useSortHandlerFn` property set in the table header object. The function passed into the `sortHandlerFn` prop sorts and returns the table data. The other columns use the default built-in client side sort function because the `useSortHandlerFn` property is not set in the header objects.
+
+<KTable
+  :fetcher="sortHandlerFnFetcher"
+  :headers="sortHandlerFnHeaders"
+  :sortHandlerFn="sortHandlerFn"
+  enable-client-sort
+/>
+
+```vue
+<template>
+  <KTable
+    :fetcher="fetcher"
+    :headers="headers"
+    :sortHandlerFn="sortHandlerFn"
+    enable-client-sort
+  />
+</template>
+<script>
```

Comment:
nit: can you add an empty line before the script tag?

Also, we should show `lang="ts"` for all the examples with script tags.

Reply:
Fixed